contributing/code/stack_trace.rst

```diff
@@ -0,0 +1,188 @@
+exception. ``composer home acme/router`` should lead you to the right
+place for that. As Symfony is a monorepository, use ``composer home
+symfony/symfony`` when reporting a bug for any component.
+
+Getting Stack Traces with Symfony
+---------------------------------
+
+Now that we have all this in mind, let us see how to get a stack trace
+with Symfony.
+
+Stack Traces in your Web Browser
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Several things need to be paid attention to when picking a stack trace
+from your development environment through a web browser:
+
+1. Are there several exceptions? If yes, the most interesting one is
+   often exception 1/n which, is shown _last_ in the example below (it
+   is the one marked as exception [1/2]).
+2. Under the "Stack Traces" tab, you will find exceptions in plain
+   text, so that you can easily share them in e.g. bug reports. Make
+   sure to **remove any sensitive information** before doing so.
+3. You may notice there is a logs tab too; this tab does not have to do
+   with stack traces, it only contains logs produced in arbitrary places
+   in your application. They may or may not relate to the exception you
+   are getting, but are not what the term "stack trace" refers to.
+
+.. image:: /_images/contributing/code/stack-trace.gif
+   :align: center
+   :class: with-browser
+
+Since stack trace may contain sensitive data, they should not be exposed
+in production. Getting a stack trace from your production environment,
+although more involving, is still possible with solutions that include
+but are not limited to sending them to an email address with monolog.
+
+Stack Traces in the CLI
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Exceptions might occur when running a Symfony command. By default, only
+the message is shown because it is often enough to understand what is
+going on:
+
+.. code-block:: terminal
+
+   $ php bin/console debug:exception
+
+
+      Command "debug:exception" is not defined.
+
+      Did you mean one of these?
+          debug:autowiring
+          debug:config
+          debug:container
+          debug:event-dispatcher
+          debug:form
+          debug:router
+          debug:translation
+          debug:twig
+
+
+If that is not the case, you can obtain a stack trace by increasing the
+`verbosity level</console/verbosity.rst>` with ``--verbose``:
+
+.. code-block:: terminal
+
+   $ php bin/console --verbose debug:exception
+
+    In Application.php line 644:
+
+      [Symfony\Component\Console\Exception\CommandNotFoundException]
+      Command "debug:exception" is not defined.
+
+      Did you mean one of these?
+          debug:autowiring
+          debug:config
+          debug:container
+          debug:event-dispatcher
+          debug:form
+          debug:router
+          debug:translation
+          debug:twig
+
+
+    Exception trace:
+      at /home/greg/dev/sf-demo/vendor/symfony/console/Application.php:644
```

Comment:
I would prefer to remove the "/greg/" part from all paths

Reply:
I will use `/app` instead of `/home/greg/dev/sf-demo`

Reply:
Thanks